app.js

```diff
@@ -22,6 +23,18 @@ app.use(session({ secret: '123456789QWERTY' }));
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
 
+// Middleware to send user info to layout
+app.use(function (req, res, next) {
```

Comment:
Don't make a db request each time you load a page. When the user is log in, put his information in a session variable.